wa-integrations/slack/routes/management.js

```diff
@@ -0,0 +1,21 @@
+const Router = require('express').Router();
+const Channel = require('../models/slackChannels');
+const Contact = require('../models/contacts');
+
+const appConfig = require('../config');
+
+Router.post("/", async function(req, resp) {
```

Comment:
Watch use of single quote everywhere to unite the code style

Reply:
and all " replaced by '